plaid/model/account_subtype.py

```diff
@@ -62,11 +62,14 @@ class AccountSubtype(ModelSimple):
             'ISA': "isa",
             'IRA': "ira",
             'LIF': "lif",
+            'LIFE_INSURANCE': "life insurance",
             'LIRA': "lira",
             'LRIF': "lrif",
             'LRSP': "lrsp",
             'NON-TAXABLE_BROKERAGE_ACCOUNT': "non-taxable brokerage account",
             'OTHER': "other",
+            'OTHER_INSURANCE': "other insurance",
+            'OTHER_ANNUITY': "other annuity",
```

Comment:
What about `ebt`? https://github.com/plaid/plaid-python/issues/350

Reply:
hello! thanks for your comment and for making me aware of that issue. i'm going to be closing out this PR as it's autogenerated, but will keep that issue in mind for next release :).